mmv1/third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -169,24 +313,71 @@ func resourceDataflowFlexTemplateJobRead(d *schema.ResourceData, meta interface{
 		return handleNotFoundError(err, d, fmt.Sprintf("Dataflow job %s", jobId))
 	}
 
+	if err := d.Set("job_id", job.Id); err != nil {
```

Comment:
Similar to previous comment: does this resource use a `job.Id` and `job.Type`? If it does, and we need to add them here, we would need to be careful that it does not introduce any breaking changes or permanent diffs

Reply:
it is set by the resourceDataflowFlexTemplateJobRead function?

Reply:
I was able to get the answers for my question above:
- The `job_id` is really only needed here for the import, and since that is being added as part of this PR, that makes sense.
- The `type` is needed for the batch jobs to properly get re-created if the users changes one of the fields.
- Neither should be breaking